src/main/webapp/app/shared/components/button.component.html

```diff
@@ -1,10 +1,11 @@
 <button
     [ngClass]="['jhi-btn', 'btn', btnType, btnSize]"
     [type]="shouldSubmit ? 'submit' : 'button'"
-    ngbTooltip="{{ tooltip | artemisTranslate }}"
+    [ngbTooltip]="tooltip | artemisTranslate"
```

Comment:
Can you use jhitranslate here.

Reply:
As far as I know, it is not possible to combine two directives without introducing a new directive that explicitly combines them.

Am I missing something here? I didn't find a syntax for that